app/views/layouts/application.html.erb

```diff
@@ -17,8 +17,7 @@
     <%= stylesheet_link_tag 'application', media: 'all' %>
     <%= javascript_pack_tag 'application' %>
 
-    <%= javascript_pack_tag 'beforeSearch' %>
```

Comment:
The javascript used to load on every page that inherited from this layout. Now it doesn't.